action-bump-version/action.yaml

```diff
@@ -0,0 +1,60 @@
+name: Bump version
+description: Bump the current version in file VERSION and open a new PR with the changes
+
+inputs:
+  current-version:
+    required: true
+    description: the current version to bump
+  branch:
+    required: true
+    description: branch name for the new PR to open
+  level:
+    description: 'what type of version to bump (major, minor, ...), default=patch'
+    default: patch
+
+runs:
+  using: "composite"
+  steps:
+    - name: Ensure git is installed
+      shell: bash
+      run: |
+        git --version
+    - name: Ensure gh is installed
+      shell: bash
+      run: |
+        gh --version
+    - name: Ensure VERSION file exists
+      shell: bash
+      run: |
+        if [ ! -f ./VERSION ]; then
+          echo "This action bumps the ${{ input.level }} version in the file ./VERSION" 1>&2
+          echo "file ./VERSION not found" 1>&2
+          exit 1
+        fi
+    - name: Get next version
+      id: semver
+      uses: actions-ecosystem/action-bump-semver@v1
+      with:
+        current_version: ${{ inputs.current-version }}
+        level: ${{ inputs.level }}
+    - name: Bump component version
+      shell: bash
+      run: |
+        echo "VERSION=${{ steps.semver.outputs.new_version }}" > ./VERSION
```

Comment:
This does not work for Federation:
```
VERSION_MAJOR=9
VERSION_MINOR=2
VERSION_PATCH=0
VERSION_HOTFIX=0
VERSION_SUFFIX=
VERSION_PREFIX=
```

Reply:
neither for Artesca. but for those with a single version in a file it works, we have 3 repos like this in storage squad so I though I make the action common here to anyone whom it may be useful to.